packages/components/src/mobile/bottom-sheet/bottom-sheet-navigation/navigation-container.native.js

```diff
@@ -94,9 +94,7 @@ function BottomSheetNavigationContainer( { children, animate, main, theme } ) {
 			}
 
 			if ( height > 1 ) {
-				if ( currentHeight === 1 ) {
-					setCurrentHeight( height );
-				} else if ( animate ) {
+				if ( animate ) {
```

Comment:
My interpretation of the original intent of this code is to avoid animating setting the height during the _initial layout_. However, it was erroneously applied when navigating _away from full-screen_ bottom sheets. Removing it resolve this issue. 

From my testing, the non-animation setting was used when a block settings is first opened, but I am unable to perceive a difference with or without its presence.